assets/src/application/ui/forms/espressoForm/context.ts

```diff
@@ -0,0 +1,10 @@
+import { createContext } from 'react';
+import { FormContextProps } from './types';
+
+const Context = createContext<FormContextProps>({});
+
+const { Provider } = Context;
+
+export const FormProvider = Provider;
+
+export default Context;
```

Comment:
cool... but can we plz move this to `assets\src\application\services\context` so that we never have to go searching for context providers? Or will this NEVER be used outside of the `application/ui/forms/espressoForm/` folder?

Reply:
I didn't want to couple it tightly with EDTR and yes, it's never going to be used outside that.

Reply:
> I didn't want to couple it tightly with EDTR

???

it wouldn't be tightly coupled to EDTR if it was in `application\services\context`

> it's never going to be used outside that.

ya ok that's fine then